eslint.config.js

```diff
@@ -0,0 +1,99 @@
+import foxkit from "eslint-config-foxkit/flat.js";
+import foxkitReact from "eslint-config-foxkit-react/flat.js";
+import prettier from "eslint-config-prettier";
+import * as astroPlugin from "eslint-plugin-astro";
+import importPlugin from "eslint-plugin-import";
+import reactPlugin from "eslint-plugin-react";
+import tsEslint from "typescript-eslint";
+
+/**
+ * @type {import("typescript-eslint").ConfigWithExtends}
+ */
+const ignorePaths = {
+  name: "ignorePaths",
+  ignores: [
+    "dist",
+    "public",
+    ".astro",
+    ".astro-cache",
```

Comment:
is this documented anywhere? afaik `.astro-cache` only used in xenia-web-next right now